generators/form/templates/IntroductionPage.jsx.ejs

```diff
@@ -1,99 +1,112 @@
-          subtitle="Equal to VA Form <%= formNumber %> (<%= appName %>)"
-        />
-        <SaveInProgressIntro
-          headingLevel={2}
-          prefillEnabled={formConfig.prefillEnabled}
-          messages={formConfig.savedFormMessages}
-          pageList={pageList}
-          startText="Start the Application"
-          devOnly={{
-            forceShowFormControls: true,
-          }}
-        >
-          Please complete the <%= formNumber %> form to apply for <%= benefitDescription %>.
-        </SaveInProgressIntro>
-        <h2 className="vads-u-font-size--h3 vad-u-margin-top--0">
-          Follow the steps below to apply for <%= benefitDescription %>.
-        </h2>
-        <va-process-list>
-          <li>
-            <h3>Prepare</h3>
-            <h4>To fill out this application, you’ll need your:</h4>
-            <ul>
-              <li>Social Security number (required)</li>
-            </ul>
-            <p>
-              <strong>What if I need help filling out my application?</strong>{' '}
-              An accredited representative, like a Veterans Service Officer
-              (VSO), can help you fill out your claim.{' '}
-              <a href="/disability-benefits/apply/help/index.html">
-                Get help filing your claim
-              </a>
-            </p>
-          </li>
-          <li>
-            <h3>Apply</h3>
-            <p>Complete this <%= benefitDescription %> form.</p>
-            <p>
-              After submitting the form, you’ll get a confirmation message.
-              You can print this for your records.
-            </p>
-          </li>
-          <li>
-            <h3>VA Review</h3>
-            <p>
-              We process claims within a week. If more than a week has passed
-              since you submitted your application and you haven’t heard back,
-              please don’t apply again. Call us at.
-            </p>
-          </li>
-          <li>
-            <h3>Decision</h3>
-            <p>
-              Once we’ve processed your claim, you’ll get a notice in the mail
-              with our decision.
-            </p>
-          </li>
-        </va-process-list>
+export const IntroductionPage = props => {
+  const userLoggedIn = useSelector(state => isLoggedIn(state));
+  const userIdVerified = useSelector(state => isLOA3(state));
+  const { route } = props;
+  const { formConfig, pageList } = route;
+  const showVerifyIdentify = userLoggedIn && !userIdVerified;
+
+  useEffect(() => {
+    scrollToTop();
+    focusElement('h1');
+  }, []);
+
+  return (
+    <article className="schemaform-intro">
+      <FormTitle title={TITLE} subTitle={SUBTITLE} />
+      <h2 className="vads-u-font-size--h3 vad-u-margin-top--0">
+        Follow the steps below to apply for <%= benefitDescription %>.
+      </h2>
+      <ProcessList />
+      {showVerifyIdentify ? (
+        <div>{/* add verify identity alert if applicable */}</div>
+      ) : (
         <SaveInProgressIntro
```

Comment:
There's a [new intro page pattern for applications](https://design.va.gov/templates/forms/introduction#examples-application):

## Unauthenticated

Save in progress component _only below_ the process list

## Authenticated

- Verify alert (if applicable) _above_ the process list (not in design, but implied)
- Save in progress component _above_ and _below_ (button only) the process list

Reply:
I think we're going to be adjusting this area some more over the next couple of weeks with new DST changes to introduction page SIP/verify alert. I've also added a new ticket [[yeoman] unauthenticated vs authenticated question ](https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team/new?issueType=issue). Do you think it's fine to leave this as is for now and continue to adjust as followups?